rollup.config.js

```diff
@@ -33,18 +34,26 @@ const tsConfig = {
 };
 const input = 'src/bson.ts';
 
-const plugins = [
-  typescript(tsConfig),
-  nodeResolve({ preferBuiltins: false }),
-  nodeBuiltins(),
-  nodeGlobals(),
-  commonjs({ extensions: ['.js', '.ts'] }),
-  babel({
-    babelHelpers: 'external',
-    plugins: ['@babel/plugin-external-helpers'],
-    presets: [['@babel/env', { modules: false }]]
-  })
-];
+const plugins = ({ browser = false }) => {
+  return [
+    typescript(tsConfig),
+    nodeResolve({ preferBuiltins: false }),
+    nodeBuiltins(),
+    nodeGlobals(),
+    replace({
+      preventAssignment: true,
+      values: {
+        'process.browser': browser
```

Comment:
Could we rename this to something else so there's no confusion between this and the Node global `process` object?

Reply:
Good point, I renamed it to `rollupProcess`